dags/utils/dataproc.py

```diff
@@ -166,6 +167,7 @@ def delete_cluster(self):
         """
         return DataprocClusterDeleteOperator(
             task_id='delete_dataproc_cluster',
+            trigger_rule=TriggerRule.ALL_DONE,
```

Comment:
This will affect more than LTV, so checking in with you @acmiyaguchi to ensure it won't break anything that expects the cluster to stick around if the upstream job fails.

Reply:
I don't think there are any jobs that rely on this behavior.